test.dockerfile

```diff
@@ -1,44 +1,68 @@
-# A Dockerfile that sets up a full Gym install
-FROM quay.io/openai/gym:base
-
-RUN apt-get update \
-    && apt-get install -y libav-tools \
-    python-numpy \
-    python-scipy \
-    python-pyglet \
-    python-setuptools \
-    libpq-dev \
-    libjpeg-dev \
-    curl \
-    cmake \
-    swig \
-    python-opengl \
-    libboost-all-dev \
-    libsdl2-dev \
-    wget \
-    unzip \
-    git \
-    xpra \
-    libav-tools  \
-    python3-dev \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* \
-    && easy_install pip
+# A Dockerfile that sets up a full Gym install with test dependencies
+FROM ubuntu:16.04
 
-WORKDIR /usr/local/gym/
-RUN mkdir -p gym && touch gym/__init__.py
-COPY ./gym/version.py ./gym/
-COPY ./requirements.txt ./
-COPY ./setup.py ./
-COPY ./tox.ini ./
-
-RUN pip install tox
-# Install the relevant dependencies. Keep printing so Travis knows we're alive.
-RUN ["bash", "-c", "( while true; do echo '.'; sleep 60; done ) & tox --notest"]
-
-# Finally, clean cached code (including dot files) and upload our actual code!
-RUN mv .tox /tmp/.tox && rm -rf .??* * && mv /tmp/.tox .tox
+ARG MUJOCO_KEY
+RUN echo $MUJOCO_KEY
+
+# Install keyboard-configuration separately to avoid travis hanging waiting for keyboard selection
+RUN \
+    apt-get -y update && \
+    apt-get install -y keyboard-configuration && \
+
+# Maybe Install python3.6 on ubuntu 16.04 ?
+#    apt-get install -y software-properties-common && \
+#    add-apt-repository -y ppa:jonathonf/python-3.6 && \
+#    apt-get -y update && \
+#    apt-get -y install python3.6 python3.6-distutils python3.6-dev
+
+    apt-get install -y \ 
+        python-setuptools \
+        python-pip \
+        libpq-dev \
+        zlib1g-dev \
+        libjpeg-dev \
+        curl \
+        cmake \
+        swig \
+        python-opengl \
+        python-numpy \
+        python-pyglet \
+        python3-opengl \
+        libboost-all-dev \
+        libsdl2-dev \
+        libosmesa6-dev \
+        patchelf \
+        wget \
+        unzip \
+        git \
+        vim \
+        xvfb \
+        ffmpeg \
+        python3-dev && \
+
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install tox && \
+
+# Download mujoco
+    mkdir /root/.mujoco && \
+    cd /root/.mujoco  && \
+    wget https://www.roboti.us/download/mjpro150_linux.zip  && \
+    unzip mjpro150_linux.zip && \
+
+# Inject the mujoco license key (if planning to publish image, this step has to be runtime)
+    echo $MUJOCO_KEY | base64 --decode > mjkey.txt && \
```

Comment:
Why have this in the Dockerfile at all? It's not harmful per-se, but might be nicer to just keep `MUJOCO_KEY` to the "docker run" invocation so we don't have to think about it.

Reply:
sounds good, I'll adjust it